src/core_atmosphere/physics/physics_noaa/SMOKE/module_add_emiss_burn.F90

```diff
@@ -0,0 +1,181 @@
+!>\file  module_add_emiss_burn.F90
+!! This file adds the biomass burning emissions to the smoke field.
+
+module module_add_emiss_burn
+!RAR: significantly modified for the new BB emissions
+  use mpas_kind_types
+  use mpas_smoke_config
+CONTAINS
+  subroutine add_emis_burn(dtstep,dz8w,rho_phy,pi,ebb_min,          &
+                           !smoke,smoke_coarse,julday,gmt,xlat,xlong,     &
+                           smoke,julday,gmt,xlat,xlong,     &
+                           fire_end_hr, peak_hr,time_int,           &
+                           coef_bb_dc, fire_hist, hwp, hwp_prevd,   &
+                           !swdown,ebb_dcycle, ebu,ebu_coarse,fire_type,&
```

Comment:
same

Reply:
The commented lines are for the two versions (MPAS-Aerosol and MPAS-Smoke/Dust).